README.md

```diff
@@ -1,3 +1,78 @@
-# skat
+# skat [![GitHub license](https://img.shields.io/github/license/Dichloromethane/skat)](https://github.com/Dichloromethane/skat/blob/master/LICENSE)
 
-Don't read me
+[![Repo Size](https://img.shields.io/github/repo-size/Dichloromethane/skat.svg)](https://github.com/Dichloromethane/skat)
+[![Release Status](https://img.shields.io/badge/status-alpa-orange)](https://github.com/Dichloromethane/skat)
+
+---
+
+**Skat** is a client-server application to play a round of Skat with multiple players.
+
+The application is programmed in **C** and therefore unbeatably **fast**. It is possible to play Skat via the **command line** or via a **graphical client**.
+
+Currently only **Linux** is supported as operating system. It is currently tested on **Debian / Ubuntu**.
+
+---
+
+## Quick Start
+
+For easy installation on Debian the following steps must be executed:
+
+```sh
+apt update
+apt install -y libglfw3-dev libfreetype-dev g++
+make all
+```
+
+After the installation the server can be started as follows:
+
+```sh
+./skat_server
+```
+
+The command line client can be executed with the following command:
+
+```sh
+./skat_client "playername"
+```
+
+---
+
+## requirements
+
+The following requirements must be met to form the project.
+
+On Debian the following packages must be installed:
+
+- libglfw3-dev
+- libfreetype-dev
+- g++
+
+---
+
+## tools
+
+It is possible to display the dependencies between the classes.
+For this purpose the script dev_graph.sh in the tools folder can be used.
+
+Requirements:
+
+- package **graphviz** on Debian
+
+Arguments:
+
+- -p _print_graph_ instead of creating an image, the structure is printed in JSON format
```

Comment:
this option does not require an argument, it is a flag. Can you make that clearer?
Also the format is called [DOT](https://en.wikipedia.org/wiki/DOT_%28graph_description_language%29).